src/main/java/com/jcabi/github/mock/MkIssueLabels.java

```diff
@@ -115,11 +119,31 @@ public Issue issue() {
     public void add(@NotNull(message = "labels can't be NULL")
         final Iterable<String> labels
     ) throws IOException {
+        final Set<String> oldlabels = this.labelSet();
```

Comment:
@cvrebert We should name this variable `existing`, as per our naming convention [here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names).